src/LoginPage.cpp

```diff
@@ -453,7 +454,7 @@ LoginPage::onLoginButtonClicked()
                         sso->deleteLater();
                 });
                 connect(sso, &SSOHandler::ssoFailed, this, [this, sso]() {
-                        emit loginError(tr("SSO login failed"));
+                        emit showError(error_label_, tr("SSO login failed"));
```

Comment:
This does not emit a signal, but calls a function, right? In that case could you remove the emit?